tests/rt.conf

```diff
@@ -61,6 +61,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp_regional,FV3_GFS_2017_gfdlmp_regiona
 RUN     | fv3_ccpp_regional_control                                                                                                      | standard    |                | fv3         |
 RUN     | fv3_ccpp_regional_restart                                                                                                      | standard    |                | fv3         | fv3_ccpp_regional_control
 RUN     | fv3_ccpp_regional_quilt                                                                                                        | standard    |                | fv3         |
+#RUN     | fv3_ccpp_regional_quilt_netcdf_parallel                                                                                        | standard    |                | fv3         |
```

Comment:
@junwang-noaa @DusanJovic-NOAA Does this mean that by default the fv3_ccpp_regional_quilt_netcdf_parallel test will not be included in the regression tests?